games/d.yaml

```diff
@@ -989,6 +989,25 @@
   - https://dreerally.com/wp-content/uploads/2016/12/Captura3-300x236.jpg
   - https://dreerally.com/wp-content/uploads/2016/12/Captura4-300x233.jpg
 
+- name: Dstroy
+  originals:
+  - Dstroy
+  type: clone
+  url: 'http://narroin.free.fr/dstroy/'
+  content: commercial
+  langs:
+  - C++
+  frameworks:
+  - SDL
+  licenses:
+  - GPL2
+  updated: '2023-02-17'
+  images:
+  - 'http://narroin.free.fr/dstroy/images/screen1.png '
+  - 'http://narroin.free.fr/dstroy/images/screen2.png '
+  - 'http://narroin.free.fr/dstroy/images/screen3.png'
+  - 'http://narroin.free.fr/dstroy/images/screen4.png'
```

Comment:
please find https images if possible; osgc is served over https so embedding http images is not possible